server/startup/migrations/v238.ts

```diff
@@ -0,0 +1,34 @@
+import { Mongo } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations/server';
+import { isEnterprise } from '../../../ee/app/license/server';
+import { Settings } from '../../../app/models/server/raw';
+// import { Banner } from '../../sdk';
+// import { BannerPlatform } from '../../../definition/IBanner';
+// import { IUser } from '../../../definition/IUser';
```

Comment:
Remove these comments

Reply:
done

Reply:
I think you haven't pushed your changes, @g-thome 😬